### PR TITLE
Small edits

### PR DESCRIPTION
**Pull request by apt105**
Mainly adding links to github pages, amazon appstore, tidying up the github readme, add details about termux, and a page on adding a button to the lora32 v2.1-1.6.

### CODE REVIEW

**Comment by apt105** [comment]
https://meshtastic-git-small-edits-apt105.vercel.app/docs/software/android/android-installation

**Comment by sachaw** [comment]
Ready to merge?

**Comment by apt105** [comment]
Yup, ready now, thanks.

File: README.md [hunk 1]
```diff
@@ -33,85 +33,11 @@
 
 ## What is Meshtastic?
 
-<table>
-<tr>
-<td>
 <p>
```
Comment (jfirwin) [inline README.md:36]: Since we've removed much of the HTML from much of the document, could we just make this portion vanilla markdown now?

Comment (jfirwin) [reply README.md:36]: Line 36+, don't know why that didn't save correctly.

Comment (apt105) [reply README.md]: Good point, html removed

**Comment by apt105** [comment]
Ready for merge again, ta

**Pull request merged**